docs/reference/query-languages/esql/_snippets/commands/types/lookup-join.md

```diff
@@ -0,0 +1,77 @@
+% This is generated by ESQL's AbstractFunctionTestCase. Do not edit it. See ../README.md for how to regenerate it.
```

Comment:
nit: this is generated from a different class, actually

Reply:
I kind of thing the increase in entropy from the fix is not worth it. Perhaps if we have more tests that do this...